python/insights_ansible_playbook_lib/serialization.py

```diff
@@ -0,0 +1,97 @@
+import typing
+
+import yaml.parser
+import yaml.scanner
+import yaml.composer
+
+
+__all__ = ["Loader", "serialize_play"]
+
+
+class CustomSafeConstructor(yaml.constructor.SafeConstructor):
+    def construct_yaml_bool(self, node: "yaml.ScalarNode"):  # type: ignore
+        value: str = self.construct_scalar(node)  # type: ignore
+        if value.lower() not in ("true", "false"):
+            return value
+        return super().construct_yaml_bool(node)
+
+    def construct_yaml_int(self, node: "yaml.ScalarNode"):  # type: ignore
+        value: str = self.construct_scalar(node)  # type: ignore
+        if ":" not in value:
+            return super().construct_yaml_int(node)
+        return value
+
+
+class Loader(
+    yaml.reader.Reader,
+    yaml.scanner.Scanner,
+    yaml.parser.Parser,
+    yaml.composer.Composer,
+    CustomSafeConstructor,
+    yaml.resolver.Resolver,
+):
+    def __init__(self, stream: str):
+        yaml.reader.Reader.__init__(self, stream)
+        yaml.scanner.Scanner.__init__(self)
+        yaml.parser.Parser.__init__(self)
+        yaml.composer.Composer.__init__(self)
+        CustomSafeConstructor.__init__(self)
+        yaml.resolver.Resolver.__init__(self)
+
+        type(self).add_constructor(
+            "tag:yaml.org,2002:bool", CustomSafeConstructor.construct_yaml_bool
+        )  # type: ignore
+        type(self).add_constructor(
+            "tag:yaml.org,2002:int", CustomSafeConstructor.construct_yaml_int
+        )  # type: ignore
+
+
+class Serializer:
+    @classmethod
+    def _obj(cls, value: typing.Any) -> str:
```

Comment:
the code in insights-core shows a warning for unknown types of `value`; should that warning be in this version of `_obj()` too?